HuygensING/timbuctoo-gui
# Upload

## How to test
- Create a dataset manually (for now)
> ⚠️  Make sure timbuctoo is fully up to date! ⚠️ 
```gql
mutation {
  createDataSet(dataSetName: "hoi") {
    dataSetId
  }
}
```
- http://localhost:3000/details/u33707283d426f900d4d33707283d426f900d4d0d__hoi
- Drag and drop boi

## Caveats
- Can't get backend to accept upload  
`{"code":500,"message":"There was an error processing your request. It has been logged (ID 3dfc72e65a5bb5b8)."}`
- Successful upload will refresh metadata in dataset, which is nice, but from a UX perspective it's kinda weird because this clears your file list because `Upload` gets unmounted.
Perhaps consider not showing spinner if there's still data available or moving upload state to redux
